### ISSUE TITLE
bad permissions on node_exporter installed from Scylla

### ISSUE THREAD

**Issue by glommer** [scylladb/scylla#6222]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

The node_exporter files as installed by our scripts have weird ownership:

```
$ ls -lh /usr/lib/scylla/Prometheus/node_exporter/node_exporter-0.17.0.linux-amd64/node_exporter
-rwxr-xr-x. 1 3434 3434 15M Nov 30  2018 /usr/lib/scylla/Prometheus/node_exporter/node_exporter-0.17.0.linux-amd64/node_exporter
```

This is because we're just keeping the whichever ownership information that came with the packaged tar. Since it is packaged by a third party, we should enforce sane ownership on it.

**Comment by glommer** [comment]
@syuu1228 can you take a look at this ?

**Comment by glommer** [comment]
python's tarfile implementation keeps the archive's ownership information when executed as root, which we do. We need to add an explicit chown step after we unpack the archive

**Comment by slivne** [comment]
We moved to using nodeexporter provided by packaging instead of tar installation - closing this

**Comment by slivne** [comment]
I thought we moved to use distributed packages - but I don;t think we have - @syuu1228 / @penberg is this still relevant ?

**Comment by syuu1228** [comment]
@slivne I don't think it affect Scylla 4.4 and master, since we moved to rpm/deb package:
```
[vagrant@localhost yum.repos.d]$ ls -la /opt/scylladb/node_exporter/node_exporter 
-rwxr-xr-x. 1 root root 14442448 Mar 22 07:27 /opt/scylladb/node_exporter/node_exporter
```

However, we use older node_exporter_install script until Scylla 4.3, it's better to fix for previous versions (and for enterprise).

**Comment by syuu1228** [comment]
Reproduced Scylla 4.3.2:
```
[vagrant@localhost yum.repos.d]$ ls -lh /opt/scylladb/Prometheus/node_exporter/node_exporter-1.0.1.linux-amd64/
total 19M
-rw-r--r--. 1 3434 3434 12K Jun 16  2020 LICENSE
-rw-r--r--. 1 3434 3434 463 Jun 16  2020 NOTICE
-rwxr-xr-x. 1 3434 3434 19M Jun 16  2020 node_exporter
```

**Comment by avikivity** [comment]
Fixed by c7781f8 (on 4.3); later versions were fixed by ebc4076fa5c51d10fd708f90e05581a8ed844771.
